cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/io/cdap/cdap/etl/common/Constants.java

```diff
@@ -36,6 +36,7 @@
   public static final String SPARK_PIPELINE_CACHING_STORAGE_LEVEL = "spark.cdap.pipeline.caching.storage.level";
   public static final String CONSOLIDATE_STAGES = "spark.cdap.pipeline.consolidate.stages";
   public static final String CACHE_FUNCTIONS = "spark.cdap.pipeline.functioncache.enable";
+  public static final String DATASET_KRYO_ENABLED = "spark.cdap.pipeline.dataset.kryo.enable";
```

Comment:
So it seems there is inconsistency in name. Should it be `spark.cdap.pipeline.dataset.xyz` or `spark.cdap.pipeline.xyz.dataset.abc`?? I would say the former is better. I suspect the aggregate one was due to legacy issue?

Reply:
I plan that as we move into Datasets we will be using this serializer in other places, not only aggregate. Basically, whenever one create a dataset one need to pass a serializer.

Reply:
Actually, I believe we are talking about same thing. It's `spark.cdap.pipeline.dataset.kryo.enable` that is `spark.cdap.pipeline.dataset.xyz`